internal/infra/push/datadestination_sql.go

```diff
@@ -323,10 +335,14 @@ func (rw *SQLRowWriter) Write(row push.Row) *push.Error {
 	for _, h := range rw.headers {
 		values = append(values, rw.dd.dialect.ConvertValue(importedRow.GetOrNil(h)))
 	}
-	log.Trace().Strs("headers", rw.headers).Msg(fmt.Sprint(values))
+	log.Trace().Strs("headers", rw.headers).Str("table", rw.table.Name()).Msg(fmt.Sprint(values))
 
 	_, err2 := rw.statement.Exec(values...)
 	if err2 != nil {
+		// reset statement after error
+		if err := rw.commit(); err != nil {
```

Comment:
Method’s name `commit` is misleading, because the only thing the method do is trying to close the statement. Does it really commit change to the database ? If not, then the method should be remamed

Reply:
This method is the same as the `close` method without constraint restoration. New version is a refactory without the `commit` method.